.github/workflows/android.yml

```diff
@@ -0,0 +1,23 @@
+name: Android CI
+
+on:
+  push:
+    branches: [ main, dev ]
+  pull_request:
+    branches: [ main, dev ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
```

Comment:
Let's use Java 11 since that'll be enforced from AGP 7.0

`java-version: '11'`

Reply:
Sure, I am changing to version 11